silo-core/test/foundry/_mocks/ERC20UpgradableMock.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity 0.8.28;
+
+import {ERC20PermitUpgradeable} from "openzeppelin5-upgradeable/token/ERC20/extensions/ERC20PermitUpgradeable.sol";
+
+contract ERC20UpgradableMock is ERC20PermitUpgradeable {
```

Comment:
why not use `MintableToken` that can do balances/allowance on demand?

Reply:
Due to the specifics of the test. I wanted to test a delegate call.